### ISSUE TITLE
[a11y] Inputs should have a label

### ISSUE THREAD

**Issue by aniamusial** [DivanteLtd/storefront-ui#723]
**Describe the bug**
Form <input> elements should have a label. As we do not want to break the styling and we do use inputs without label, IMO we can go for ` aria-label`

https://dequeuniversity.com/rules/axe/3.4/label?application=axeAPI
